src/backends/gr.jl

```diff
@@ -244,19 +244,19 @@ function gr_polyline3d(x, y, z, func = GR.polyline3d)
     end
 end
 
+has_latex_inline_math(s::AbstractString) = occursin(r".*\$[^\$]+?\$.*", s) ≢ nothing
```

Comment:
maybe just `!==` as  `≢ ` does not display well in many fonts?

Reply:
You're right, habits :shrug:, fixed.